nmodl/__init__.py

```diff
@@ -1,4 +1,10 @@
-from ._nmodl import NmodlDriver, to_json, to_nmodl  # noqa
-from ._nmodl import __version__
+import imp
+try:
+    # check if nmodl binds are built before importing
+    imp.find_module('_nmodl')
+    from ._nmodl import NmodlDriver, to_json, to_nmodl  # noqa
+    from ._nmodl import __version__
+    __all__ = ["NmodlDriver", "to_json", "to_nmodl"]
+except ImportError:
+    print("[NMODL] [warning] :: Python bindings are not available")
```

Comment:
do we want to specify which import is failing?

Reply:
there is only one module called `_nmodl`. So IMO it's sufficient to say bindings are not enabled.